_scripts/package.json

```diff
@@ -0,0 +1,11 @@
+{
+    "private": true,
+    "name": "scripts",
+    "version": "1.0.0",
+    "description": "Test conversion script",
+    "scripts": {
+        "build": "npx tsc -p .",
+        "runjs": "node dist/convertFourslash.js",
+        "run": "npx tsc -p . && node dist/convertFourslash.js"
```

Comment:
I think I'd prefer if this code were moved into a subdir within the fourslash package, similarly to the LSP generation script, rather than introducing another top-level directory. You can also just use node's type stripping (or mjs), and avoid package.json or any deps entirely. See also the API tests.